src/register-org/components/organisation-services-access/organisation-services-access.component.html

```diff
@@ -7,12 +7,12 @@
       <form [formGroup]="servicesFormGroup">
         <div class="govuk-form-group">
           <fieldset class="govuk-fieldset" aria-describedby="changed-name-hint">
-            <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
-              <h1 class="govuk-fieldset__heading">
+            <legend>
+              <h1 class="govuk-heading-xl">
```

Comment:
@johnbenjamin-hmcts - this should be reverted once the above change is made

Reply:
@Daniel-Lam - I have changed the code so that the header and spacing match with other pages.